specs/v1.yaml

```diff
@@ -1257,6 +1394,93 @@ components:
           example: direct
         address:
           $ref: "#/components/schemas/GrantAddress"
+    RecurringGrant:
+      type: object
+      description: |-
+        A RecurringGrant represents a successfully initiated recurring grant request with a Donor Advised Fund. 
+        RecurringGrants are created when a person interacts with an instance of Chariot Connect and successfully submits a recurring grant and completes the workflow. 
+        On creation, each RecurringGrant object will also have a Grant object created to represent the first grant of the recurring grant.
+        There can be many recurring grants associated with a Connect object and therefore a Nonprofit.
```

Comment:
not sure this last sentence adds value to the description of this API and is relevant. I think it makes more sense to say: "There can be many grants associated with the same recurring grant."